terraform/schema_14110.yaml

```diff
@@ -708,16 +708,16 @@ variables:
       compartmentId: ${network_compartment_id}
       vcnId: ${wls_existing_vcn_id}
 
-  #Network Validation script execution acknowledgement
-  existing_network_validated:
-    type: string
+  skip_network_validation:
+    type: boolean
+    default: false
     required: true
-    title: "Acknowledge if Existing Network is Validated"
-    description: "Enter YES to confirm if existing Virtual Cloud Network you selected is validated using the network validation script. See <a target=\"_blank\" href=\"https://docs.oracle.com/pls/topic/lookup?ctx=en/cloud/paas/weblogic-cloud/user&id=oci_network_validate\">Validate Existing Network Setup</a>"
-    pattern: "^[yY][eE][sS]$"
+    title: "Skip Network Validation"
+    description: "Network validation script to run for existing subnets"
```

Comment:
I think the description can be improved. This description looks like this field is used to specify the script.

Reply:
changed it to "Skip running network validation script for existing subnets". I will also check with Vanaja.